dali/operators/geometry/affine_transforms/transform_crop.cc

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/geometry/affine_transforms/transform_base_op.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(TransformCrop)
+  .DocStr(R"code(Produces an affine transform matrix that maps a reference coordinate space to another one.
+
+This transform can be used to adjust coordinates after a crop operation so that a ``from_start`` point will
+be mapped to ``to_start`` and ``from_end`` will be mapped to ``to_end``.
+
+If another transform matrix is passed as an input, the operator applies the transformation to the matrix provided.
+
+.. note::
+    The output of this operator can be fed directly to the ``MT`` argument of ``CoordTransform`` operator.
+)code")
+  .AddArg(
+    "from_start",
+    R"code(The lower bound of the original coordinate space.)code",
+    DALI_FLOAT_VEC, true)
+  .AddArg(
+    "from_end",
+    R"code(The upper bound of the original coordinate space.)code",
+    DALI_FLOAT_VEC, true)
+  .AddArg(
+    "to_start",
+    R"code(The lower bound of the destination coordinate space.)code",
+    DALI_FLOAT_VEC, true)
+  .AddArg(
+    "to_end",
+    R"code(The upper bound of the destination coordinate space.)code",
+    DALI_FLOAT_VEC, true)
+  .AddOptionalArg(
+    "absolute",
+    R"code(If set to true, start and end coordinates will be swapped if start > end.)code",
+    false, true)
```

Comment:
I think this shouldn't be a run-time parameter.
```suggestion
    false, false)
```